airbyte-integrations/bases/base-normalization/build.gradle

```diff
@@ -38,26 +38,26 @@ generate.configure {
 tasks.named('check').configure {
     dependsOn generate
 }
-
-[
-        'bigquery',
-        'mysql',
-        'postgres',
-        'redshift',
-        'snowflake',
-        'oracle',
-        'mssql',
-        'clickhouse',
-        'tidb',
-        'duckdb',
-].each {destinationName ->
-    def integrationTestPython = tasks.named('integrationTestPython')
-    integrationTestPython.configure {
-        dependsOn project(":airbyte-integrations:connectors:destination-$destinationName").tasks.named('assemble')
-    }
-    // Not really sure what this task does differently from customIntegrationTestPython,
-    // but it seems to also run integration tests and as such it depends on the docker images.
-    integrationTestPython.configure {
-        dependsOn project(":airbyte-integrations:connectors:destination-$destinationName").tasks.named('assemble')
-    }
-}
+//
+//[
+//        'bigquery',
+//        'mysql',
+//        'postgres',
+//        'redshift',
+//        'snowflake',
+//        'oracle',
+//        'mssql',
+//        'clickhouse',
+//        'tidb',
+//        'duckdb',
+//].each {destinationName ->
+//    def integrationTestPython = tasks.named('integrationTestPython')
+//    integrationTestPython.configure {
+//        dependsOn project(":airbyte-integrations:connectors:destination-$destinationName").tasks.named('assemble')
+//    }
+//    // Not really sure what this task does differently from customIntegrationTestPython,
+//    // but it seems to also run integration tests and as such it depends on the docker images.
+//    integrationTestPython.configure {
+//        dependsOn project(":airbyte-integrations:connectors:destination-$destinationName").tasks.named('assemble')
+//    }
+//}
```

Comment:
@postamar as discussed, this is the sort of stuff i am ?? as to whether it should still live in gradle

Reply:
This is probably for the 
```
// python is required by the root project to run CAT tests for connectors
```
part i added back in?

Reply:
We need to check with @airbytehq/destinations as they own the `base-normalization` project and it's one of those legacy python-in-gradle projects. I've heard it's slated to go away, as well.